### PR TITLE
run "typos --write-changes" 

### PR DESCRIPTION
**Pull request by eriksjolund**
Just a test for demo purpose. Trying out https://github.com/crate-ci/typos/

Software version used
https://github.com/crate-ci/typos/releases/tag/v1.13.10

The binary was downloaded from
https://github.com/crate-ci/typos/releases/download/v1.13.10/typos-v1.13.10-x86_64-unknown-linux-musl.tar.gz

Command that was run:

```
typos --write-changes  docs cmd cni contrib dependencies docs hack libpod pkg utils
```

Signed-off-by: Erik Sjölund <erik.sjolund@gmail.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by eriksjolund** [comment]
There are some false positives. I'll remove them later.

**Comment by eriksjolund** [comment]
False positives now removed.

**Comment by baude** [comment]
/approve
LGTM

File: docs/source/markdown/podman-network.1.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ so networks have to be created again after a backend change.
 | prune      | [podman-network-prune(1)](podman-network-prune.1.md)           | Remove all unused networks                                      |
 | reload     | [podman-network-reload(1)](podman-network-reload.1.md)         | Reload network configuration for containers                     |
 | rm         | [podman-network-rm(1)](podman-network-rm.1.md)                 | Remove one or more networks                                     |
-| update     | [podman-network-upate(1)](podman-network-update.1.md)          | Update an existing Podman network                               |
+| update     | [podman-network-update(1)](podman-network-update.1.md)          | Update an existing Podman network                               |
```
Comment (edsantiago) [inline docs/source/markdown/podman-network.1.md:36]: This is something I wish I could check for in `xref-helpmsgs-manpages` but there are too many (valid) inconsistencies in subcommand names and man page paths. Thank you for catching this one. LGTM.

Comment (edsantiago) [reply docs/source/markdown/podman-network.1.md:36]: Actually, we _can_ crosscheck. #17486 will prevent this sort of mistake from happening again. Thank you for the inspiration.

**Comment by edsantiago** [comment]
/hold cancel

Thanks again!

**Pull request merged**